packages/url/src/test/index.test.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Looks good, I'd love to start using Jest here. We may have some QUnit tests to move over if we move any of the files we have tests for into modules.